keps/sig-auth/3766-referential-authorization/README.md

```diff
@@ -881,7 +900,7 @@ Users may also create ReferenceGrants to enable cross-namespace references.
 ###### Will enabling / using this feature result in introducing new API types?
 
 API Type: ReferenceGrant
-Supported Number of Objects per Cluster: No limit 
+Supported Number of Objects per Cluster: No limit
```

Comment:
This made me smile, good that we are optimistic 😄 #nolimits

Reply:
Haha whoops, updated to add more realistic goals here.